README.md

```diff
@@ -66,6 +66,12 @@ Here we are adding mint.com to the password store, but more specifically to the
 $ passgo insert mney/mint.com
 ```
 
+#### passgo insertfile group/file-name filepath
+Adding a file works almost the same as insert. Instead it has an extra argument. The file that you want to add to your vault is the final argument. Grouping works the same way with `insertfile` as `insert`.
+```
```

Comment:
Insert a blank line before this line for consistency (look at the section above). It should look like this:

    Adding a file works almost the same as insert. Instead it has an extra argument. ...

    ```
    $ passgo insertfile money/moneyfile.txt expenses.txt
    ```